docs/source/file_read/file_read.rst

```diff
@@ -0,0 +1,30 @@
+Reading NWB Files
```

Comment:
It would be useful to add anchors for all main headings in this PR to make sure we can easily reference sections.

```suggestion
.. _reading_nwb_files:

Reading NWB Files
```

Reply:
Implemented with https://github.com/NeurodataWithoutBorders/nwb-overview/pull/17/commits/2590d84fd558c445df8e7abbb8846063c9477132